easybuild/easyconfigs/t/TRIQS/TRIQS-2.1.1-foss-2018b.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS'
+version = '2.1.1'
+
+homepage = 'https://triqs.github.io/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+"""
+
+docurls = ['https://triqs.github.io/triqs/2.1.x/reference.html']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['%(version)s.tar.gz']
+source_urls = ['https://github.com/TRIQS/triqs/archive/']
+
+dependencies = [
+  ('Python', '2.7.15'),
+  ('Boost', '1.68.0', '-Python-2.7.15'),
```

Comment:
If `Python` is included as a dependency, please indicate so by using the following `versionsuffix` (right below where you define `version`), so:

```python
version = '2.1.1'
versionsuffix = '-Python-%(pyver)s'
```

The `%(pyver)s` part is a template value, which gets automatically replaced with the Python version included as a dependency.

Any dependencies that have the same versionsuffix can then leverage `versionsuffix`, so:

```python
dependencies = [
    ('Python', '2.7.15'),
    ('Boost', '1.68.0', versionsuffix),
    ...
```

In general, you should avoid hardcoding things like the `Python` version in multiple places: just specify the version once in `dependencies` (like you did above), and then use the template values like `%(pyver)s` and `%(pyshortver)s`, see https://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#avail-easyconfig-templates